KarhooSDK/Api/DataStore/User/Credentials.swift

```diff
@@ -8,21 +8,37 @@
 
 import Foundation
 
-struct Credentials {
+public struct Credentials {
     let accessToken: String
     let expiryDate: Date?
     let refreshToken: String?
-    
-    init(accessToken: String, expiryDate: Date?, refreshToken: String?) {
+    let refreshTokenExpiryDate: Date?
+
+    init(
+        accessToken: String,
+        expiryDate: Date?,
+        refreshToken: String?,
+        refreshTokenExpiryDate: Date?
+    ) {
         self.accessToken = accessToken
         self.expiryDate = expiryDate
         self.refreshToken = refreshToken
+        self.refreshTokenExpiryDate = refreshTokenExpiryDate
     }
 
-    init(accessToken: String, expiresIn: TimeInterval, refreshToken: String?) {
+    init(
```

Comment:
needs to be public in order for the DP to instantiate the Credentials we want them to pass on to us.